secret_hitler.py

```diff
@@ -1026,6 +1034,51 @@ def handle_message(self, chat_id, from_player, command, args=""):
                         return "Successfully changed name to '{}'".format(new_name)
             else:
                 return "Must be in game to change nickname"
+        elif command == "claim":
+            if from_player in self.players:
+                if args == "" or args in ["FFF", "FFL", "FLF", "LFF", "FLL", "LFL", "LLF", "LLL", "FF", "FL", "LF", "LL"]:
```

Comment:
is args always uppercase, because of the code or the does the user need to provide an uppercase argument?

Reply:
User needs to provide an uppercase argument. I thought about converting the argument to uppercase first, but since it is standard notation to use uppercase letters and I plan to hide the `/claim` within a replyMarkup button anyways, I didn’t do it here…